libass/ass_render.c

```diff
@@ -1297,7 +1296,11 @@ get_bitmap_glyph(ASS_Renderer *render_priv, GlyphInfo *info,
         ass_cache_dec_ref(info->outline);
         return;
     }
-    info->image = ass_cache_get(render_priv->cache.bitmap_cache, &key, render_priv);
+    info->bm = ass_cache_get(render_priv->cache.bitmap_cache, &key, render_priv);
+    if (!info->bm || !info->bm->buffer) {
+        ass_cache_dec_ref(info->bm);
+        info->bm = NULL;
+    }
     *pos_o = *pos;
```

Comment:
This line seems redundant now.

Reply:
Correction: this line seems redundant after this commit _and_ [this](https://github.com/libass/libass/commit/926fae558d001ab8bc6d3837c5a6a220754a385b#diff-0348e387624211dd40e575fe75ab8596R1392) hunk, which I’d like to get squashed into this commit anyway.

Reply:
Never mind: this line isn’t redundant because there are two `return`s on failure below before `*pos_o` is rewritten. Sorry for the notification noise.

If you go with setting these positions to zero on failure, I’d replace this with a zero assignment. Maybe it’s clearer to move it down to just before those `return` statements, but maybe not.